### DISCUSSION TITLE
Github account - not able to login - missed to verify email address

### DISCUSSION THREAD

**Discussion by shiddugmail** [community/community#37868] [Other Feature Feedback, Questions, & Ideas]
Hello Everyone, 

I had signed up with an email address and received the verify email address email.  For some reason I could not verify in 24 hours.

Now I cannot login because the email address is not verified. 

If I sign up again with same mail id, it says invalid or already registered email id. 

please guide what is the way to use the same email address and get through.

**Comment by jfullstackdev** [comment]
maybe Support can help you

https://support.github.com/request

**Reply by shiddugmail** [reply]
Sure thank you. I will checkup now

**Reply by shiddugmail** [reply]
While writing to support I tried the virtual assistant - it could not help me. 
I have written to the Support team. 

in the mean time in case anyone else knows answer for this please let me know.

Thank you.

**Comment by ossanna16** [comment]
Hi there @shiddugmail and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by shiddugmail** [comment]
Good morning!

Please help me understand - I didn't get any response from Github support for following ticket

> Neither able signup nor sign in #1854155
>  Open	Shiddu opened this ticket for shiddugmail 3 days ago · 2 comments

Thank you,
Regards,
Shiddu

**Reply by radar-blip** [reply]
Sometimes the support team is very busy, and it takes time for them to get back to you.
They help out on a first come, first serve basis.
Hang in there!

**Reply by shiddugmail** [reply]
Yeah sure. I will.
Not sure if it normal - no one else has a solution.
